plugin/pkg/admission/namespace/exists/admission.go

```diff
@@ -85,7 +85,7 @@ func NewExists(c client.Interface) admission.Interface {
 	reflector := cache.NewReflector(
 		&cache.ListWatch{
 			ListFunc: func() (runtime.Object, error) {
-				return c.Namespaces().List(labels.Everything())
+				return c.Namespaces().List(labels.Everything(), labels.Everything())
```

Comment:
It's a little confusing in the current state.  I am trying to find a current client that takes a fields.Selector as input.  Pod currently takes a label.Selector for both field and labels.  Is there a separate PR you are aware of that was looking to convert all of the instances of the old behavior ?